Makefile

```diff
@@ -9,3 +9,9 @@ include tools/tools.mk
 win32-background-launcher:
 	GOOS=windows go generate
 	GOOS=windows go build -ldflags -H=windowsgui -o bin/win32-background-launcher.exe ./
+	
+.PHONY: vendor
+vendor:
+	go mod tidy
+	go mod vendor
+	cd $(TOOLS_DIR) && go mod tidy && go mod vendor
```

Comment:
Maybe this could be a `vendor-tools` target in `tools/tools.mk` to keep everything self-contained?